src/test/ui/consts/ice-zst-static-access.rs

```diff
@@ -0,0 +1,32 @@
+// check-pass
```

Comment:
Are we sure "check" is enough?

Reply:
Yes, the const prop one occurred during mir optimization of `fn new`, which will still happen in check mode, and the static one happened during evaluation of the static's initializer, which also happens in check mode.